packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -163,11 +167,64 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
       from: fromAddress,
     },
   ])
+  const [initializerAbiRG, transferImplOwnershipAbiRG] = checkRGImplementationAbi(
+    ReleaseGold,
+    'ReleaseGold'
+  )
   console.info('  Deploying ReleaseGoldProxy...')
   const releaseGoldProxy = await retryTx(ReleaseGoldProxy.new, [{ from: fromAddress }])
   console.info('  Deploying ReleaseGold...')
   const releaseGoldInstance = await retryTx(ReleaseGold.new, [{ from: fromAddress }])
 
+  // We need to fund the RG implementation instance in order to initialize it.
+  await retryTx(web3.eth.sendTransaction, [
+    {
+      from: fromAddress,
+      value: web3.utils.toWei('1', 'ether'),
```

Comment:
You probably don't want to give our 10^18 as right now, but only 1 right?

Reply:
Good call